doc/build-unix.md

```diff
@@ -85,19 +85,15 @@ Now, you can either build from self-compiled [depends](/depends/README.md) or in
 
     sudo apt-get install libevent-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-test-dev
 
-Berkeley DB is required for the wallet.
-
-Ubuntu and Debian have their own `libdb-dev` and `libdb++-dev` packages, but these will install
-Berkeley DB 5.1 or later. This will break binary wallet compatibility with the distributed executables, which
-are based on BerkeleyDB 4.8. If you do not care about wallet compatibility,
-pass `--with-incompatible-bdb` to configure.
-
-Otherwise, you can build Berkeley DB [yourself](#berkeley-db).
-
 SQLite is required for the descriptor wallet:
 
     sudo apt install libsqlite3-dev
 
+Berkeley DB is required for the legacy wallet. Ubuntu and Debian have their own `libdb-dev` and `libdb++-dev` packages,
```

Comment:
nit: s/"Ubuntu and Debian"/"Debian-based systems"/

Reply:
s/"Ubuntu and Debian"/"Debian, Ubuntu and other Debian-based systems"/